code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,140 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	w_type = RECYK_WOOD
+	frame_material = /obj/item/stack/sheet/wood
+	sheets_refunded = 2
+	autoignition_temperature = AUTOIGNITION_WOOD
+	var/obj/item/held_item
+
+/obj/item/mounted/frame/trophy_mount/Destroy()
+	if(held_item)
+		qdel(held_item)
+		held_item = null
+	..()
+
+/obj/item/mounted/frame/trophy_mount/update_icon()
+	if(!held_item)
+		overlays.len = 0
+		name = initial(name)
+		desc = initial(desc)
+		return
+	name = "[held_item.name]"
+	desc = "\A [held_item] mounted on a wooden trophy mount for display."
+	var/datum/log/L = new
+	held_item.examine(L)
+	desc += "\n[L.log]"
```

Comment:
Seems like this would be better accomplished by just calling `held_item`'s `examine()` inside the trophy mount's `examine()`. That way clocks wouldn't be stopped, etc.